gulpfile.js

```diff
@@ -153,31 +179,18 @@ gulp.task('test-react-warnings', function() {
     gutil.log(gutil.colors.red.bold(message));
   };
 
-  return new IndexFileStream(require('./lib/index-static.jsx'))
+  return createIndexFileStream()
     .on('end', function() {
       console.warn = oldWarn;
       if (warnings) {
         this.emit('error', new Error('At least one warning was logged.'));
       }
     })
-    .pipe(gulp.dest('./dist'));
+    .on('data', function() {});
```

Comment:
Hey @toolness , could you help me understand why `on('data', function() {})` is needed here?